3.7/graphs-satellite-graphs.md

```diff
@@ -12,116 +12,120 @@ SatelliteGraphs are only available in the
+and replication properties. The properties `distributeShardsLike`, `replicationFactor` and
+`numberOfShards` will be set automatically. 
+
+To create a SatelliteGraph in arangosh, use the `satelliteGraph` module:
 
 {% arangoshexample examplevar="examplevar" script="script" result="result" %}
-    @startDocuBlockInline satelliteGraphExplain3_cluster
-    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphExplain3_cluster}
-    ~var graphModule = require("@arangodb/general-graph");
-    ~var satelliteGraphModule = require("@arangodb/satellite-graph");
-    ~graphModule._create("normalGraph", [ graphModule._relation("edges", "vertices", "vertices") ], [], {});
-    ~satelliteGraphModule._create("satelliteGraph", [ satelliteGraphModule._relation("satEdges", "satVertices", "satVertices") ], [], {});
-    ~db._create("collection", {numberOfShards: 8});
-    db._explain(`FOR doc in collection FOR v,e,p IN OUTBOUND "vertices/start" GRAPH "satelliteGraph" RETURN [doc,v,e,p]`, {}, {colors: false});
-    ~db._drop("collection");
+    @startDocuBlockInline satelliteGraphCreate1_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{satelliteGraphCreate1_cluster}
+    var satelliteGraphModule = require("@arangodb/satellite-graph");
+    var graph = satelliteGraphModule._create("satelliteGraph");
+    satelliteGraphModule._graph("satelliteGraph");
     ~satelliteGraphModule._drop("satelliteGraph", true);
-    ~graphModule._drop("normalGraph", true);
     @END_EXAMPLE_ARANGOSH_OUTPUT
-    @endDocuBlock satelliteGraphExplain3_cluster
+    @endDocuBlock satelliteGraphCreate1_cluster
 {% endarangoshexample %}
 {% include arangoshexample.html id=examplevar script=script result=result %}
 
-Note that now the `TraversalNode` is executed on each DB-Server, leading to a
-great reduction in required network communication, and hence potential gains
-in query performance.
+### Add vertex collections
+This is analogous to GeneralGraphs. Unlike with GeneralGraphs, there are rules for collections.
+Only collections which do not violate the rules are allowed to be added (Details in section: SatelliteGraph in detail).
+Using the SatelliteGraph module to add new collections, the module will do all the configuration automatically: 
+
+```
+arangosh> let graph = g_module._graph("aSatelliteGraph");
+arangosh> graph._addVertexCollection("aVertexCollection");
+```
 
-Caveats
--------
+### Define relations on the SatelliteGraph
+Adding edge collections works the same as with GeneralGraphs, but again, the collections are created by the SatelliteGraph module to fit the collection rules here as well.
 
-The cluster will automatically distribute all SatelliteGraphs to all
-DB-Servers, and apply synchronous replication. This means that writes are
-executed on the leader only, and then have to be replicated to all followers,
-leading to degradation in write performance.
+```
+arangosh> let graph = g_module._graph("aSatelliteGraph");
```

Comment:
Why is this not an example like the others?

Reply:
ups, because no reason :P I'll add that